modules/ecs_instances/templates/user_data.sh

```diff
@@ -61,7 +58,7 @@ EOF
 # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html
 region=$(curl 169.254.169.254/latest/meta-data/placement/availability-zone | sed s'/.$//')
 # Replace the logging region specified by our region env var with the region where the container instance is located.
-sed -i -e "s/region = us-east-1/region = $region/g" /etc/awslogs/awscli.conf
```

Comment:
I just realized we are replacing `us-east-1` here, rather than the currently specified region, `eu-west-1`.

From here: https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/QuickStartEC2Instance.html, I realize that `/etc/awslogs/awscli.conf` points to the `us-east-1` region by default. I will revert this line change back to `us-east-1`.

Reply:
Updated.